R/flex_dashboard.R

```diff
@@ -354,7 +354,6 @@ flex_dashboard <- function(fig_width = 6.0,
     # dashboard init script
     dashboardInitScript <- c(
        '<script type="text/javascript">',
-       '$(document).ready(function () {',
```

Comment:
I feel like the document needs to be loaded before we run FlexDashboard.init, but perhaps your experiments prove otherwise? In any case I think this qualifies as a  very high risk change so if there is any other way to do it that would be preferred.

Reply:
👍

Reply:
I'm okay w/ this change if you've tested it on a bunch of static, runtime: shiny, and runtime: shiny_prerendered dashboards (including dashboards that use a variety of popular htmlwidgets, whose sizing behavior might be affected by this change).